docs/pages/versions/unversioned/sdk/dataMigrationHelper.md

```diff
@@ -0,0 +1,79 @@
+---
+title: DataMigrationHelper
+---
+
+**`DataMigrationHelper`** helps you to solve a problem with missing files after SDK upgrade on Android.
+The problem only occurs when the ejected/standalone application is upgraded from SDK 32 or below to SDK 33 or above.
+
```

Comment:
I think we need to provide a clear and straighforward section like this, otherwise this will just be confusing to users.
```suggestion

## How to use

- **If you are upgrading your standalone app from SDK 32 or below** and use the `FileSystem` API, we recommend running `DataMigrationHelper.migrateFilesFromLegacyDirectoryAsync()` as early as possible in your app's lifecycle to properly migrate old files. After this, you should be good to go.
- **If you have already upgraded your standalone app from SDK 32 or below to SDKs 33-35 and are upgrading again**, and use the `FileSystem` API, we recommend calling `DataMigrationHelper.migrateFilesFromLegacyDirectoryAsync(DataMigrationHelper.noopResolve)` as early as possible in your app's lifecycle. This will migrate files from the legacy directory only if files with the same name do not already exist in the new location -- allowing you migrate old users without fear of overwriting newly created data for newer users.
- **If your app never used the `FileSystem` API in SDK 32 or below**, there is no need for you to use this module.

```